config/personality/es.ini

```diff
@@ -31,18 +32,50 @@ apps_add =
   com.endlessm.howto.es
   com.endlessm.maternity.es
   com.endlessm.math.es
+  com.endlessm.microenterprises.es_GT
   com.endlessm.myths.es
   com.endlessm.physics.es
+  com.endlessm.programming
   com.endlessm.programming_guide.es
   com.endlessm.soccer.es
+  com.endlessm.social_enterprises.es_GT
   com.endlessm.socialsciences.es
   com.endlessm.travel.es
+  com.endlessm.video_animal_kingdom
+  com.endlessm.vroom.es
   com.endlessm.water_and_sanitation.es
   com.endlessm.world_literature.es
+  com.endlessnetwork.arduinoprojects
+  com.endlessnetwork.blendertutorials
+  com.endlessnetwork.csstutorials
+  com.endlessnetwork.htmltutorials
+  com.endlessnetwork.sciencesnacks
 
 [flatpak-remote-flathub]
 apps_add =
   ar.com.pilas_engine.App
+  as.may.moat
+  com.github.carlos157oliveira.Calculus
+  com.google.AndroidStudio
```

Comment:
I don't think this is redistributable:

https://github.com/flathub/com.google.AndroidStudio/blob/master/com.google.AndroidStudio.json#L43-L52

Reply:
Yes.

Reply:
Thanks for your reminder.  I dropped `com.google.AndroidStudio` and `com.visualstudio.code` from the list in the new commit.